signals.mss

```diff
@@ -671,27 +671,12 @@ Format details:
       text-face-name: @bold-fonts;
       text-size: 10;
     }
-  }
 
-  /***********************************************/
-  /* NL ETCS stopplaatsmarkering 227b (driehoek) */
-  /***********************************************/
-  ["feature"="NL:227b"]["train_protection_form"="sign"]["train_protection_type"="block_marker"]["shape"="triangle"] {
+    /* triangle-shaped stop marker */
+    ["shape"="triangle"] {
```

Comment:
Please also select ```["feature"="NL:227b"]``` to prevent someone (ab)using this to display unsupported combinations with that signal.

Reply:
Implemented